src/index.js

```diff
@@ -175,12 +178,23 @@ function start() {
       close(); // close the app for Linux and Windows
     }
   });
+
+  // Check if app is installed with Squirrel
+  if ((OS === 'win32' && fs.existsSync(path.resolve(path.dirname(process.execPath), '..', 'update.exe')))
```

Comment:
I removed the `isDev` check because the update can only happen if on Windows the app was installed through `Squirrel.Windows`. For macOS, the app needs to be signed for it to be updated, so we check that the app was started from its own process rather than `electron`. When running as `isDev`, both of those conditions won't let the updater run anyway.